src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = sourceString
+    .split(';')
+    .filter((style) => style.includes(':'))
+    .reduce((prev, style) => {
+      const [key, value] = style.split(':');
+
+      if (key && value) {
+        prev[key.trim()] = value.trim();
+      }
+
+      return prev;
```

Comment:
The 'reduce' method is used appropriately to transform the array of strings into an object of CSS properties. Make sure to handle edge cases where there may be extra whitespace or other characters that could affect the parsing of keys and values.